### PR TITLE
Add deviation for interface counter 

### PR DESCRIPTION
**Pull request by mojiiba**
This pull adds a deviation to use gnmi get at the container level for reading interface counters. It also guards the ipv4/enabled with the related deviation.

### CODE REVIEW

File: internal/deviations/deviations.go [hunk 1]
```diff
@@ -70,6 +70,8 @@ var (
 	InterfaceEnabled = flag.Bool("deviation_interface_enabled", false,
 		"Device requires interface enabled leaf booleans to be explicitly set to true.  Full OpenConfig compliant devices should pass both with and without this deviation.")
 
+	CounterLeavesNotUpdated = flag.Bool("deviation_counters_leaves_notupdated", false, "Device updates counters when they are queried at the container level.")
```
Comment (liulk) [inline internal/deviations/deviations.go:73]:
```suggestion
	InterfaceCountersFromContainer = flag.Bool("deviation_interface_counters_from_container", false, "Device only supports querying interface counters from the state container, not from individual counter leaves.")
```

File: feature/gnmi/ate_tests/telemetry_basic_check_test/telemetry_basic_check_test.go [hunk 2]
```diff
@@ -897,7 +906,9 @@ func ConfigureDUTIntf(t *testing.T, dut *ondatra.DUTDevice) {
 		}
 		i.GetOrCreateEthernet()
 		s := i.GetOrCreateSubinterface(0).GetOrCreateIpv4()
-		s.Enabled = ygot.Bool(true)
+		if *deviations.InterfaceEnabled {
+			s.Enabled = ygot.Bool(true)
+		}
```
Comment (liulk) [inline feature/gnmi/ate_tests/telemetry_basic_check_test/telemetry_basic_check_test.go:911]: Already done in #641 ?

**Review by liulk — CHANGES_REQUESTED** [review]

File: feature/gnmi/ate_tests/telemetry_basic_check_test/telemetry_basic_check_test.go (hunk 2 above)
Comment (mojiiba) [reply feature/gnmi/ate_tests/telemetry_basic_check_test/telemetry_basic_check_test.go:911]: no, I missed this one

File: internal/deviations/deviations.go (hunk 1 above)
Comment (mojiiba) [reply internal/deviations/deviations.go]: thank you, addressed

**Review by liulk — APPROVED** [review]

**Pull request merged**